Runtime/Scripts/RGBotLocalRuntime/RG.cs

```diff
@@ -61,15 +61,12 @@ public List<RGStateEntity> GetMyPlayers()
         /**
          * <summary>Used to find the closest Entity to the given position.</summary>
          * <param name="objectType">{string | null} Search for entities of a specific type</param>
-         * <param name="position">
-         *     {Vector3 | null} Position to search from.  If not passed, attempts to use the client's bot
-         *     position in index 0.
-         * </param>
+         * <param name="position">{Vector3} Position to search from</param>
          * <param name="filterFunction">{Func&lt;RGStateEntity, bool&gt; | null} Function to filter entities.</param>
          * <returns>{RGStateEntity} The closest Entity matching the search criteria, or null if none match.</returns>
          */
         [CanBeNull]
-        public RGStateEntity FindNearestEntity(string objectType = null, Vector3? position = null,
+        public RGStateEntity FindNearestEntity(Vector3 position, string objectType = null,
```

Comment:
This should still allow a null position. This method is a helper for users to find entities from their botcode based on their distance from the specified position. null will default to searching from the bot's current position, but I can also search for the closest entity to some other position - whether that's a GameObject's or some arbitrary waypoint.

Reply:
I have changed the function back. There are just a couple of adjustments to handle the always present position vectors